src/plugins/bar-graph/bar-graph.scss

```diff
@@ -75,23 +78,77 @@
       margin: 5px 0 5px 5px;
       align-items: center;
 
+      button, button.chakra-menu__menu-button {
+        background: transparent;
+        border: none;
+        border-radius: 0;
+        height: 26px;
+        margin: 0 7px 0 0;
+        width: 26px;
+
+        &> span { min-width: 26px; }
+      }
+
       .color-button {
+        cursor: pointer;
         width: 26px;
         height: 26px;
-        margin: 0 7px 0 0;
+        margin: 0;
         padding: 5px;
         background-color: $workspace-teal-light-8;
         border-radius: 5px;
-        border: solid 1.5px #949494;
+        border: solid 1.5px $charcoal-light-1;
+        pointer-events: auto;
+
+        &:hover {
+          background: $color-button-hover;
+        }
+        &:active, &:focus {
+          background: $color-button-active;
+        }
 
         .color-swatch {
           border: 1px solid black;
+          outline: 2px solid white;
           width: 100%;
           height: 100%;
           margin: 0;
         }
       }
 
+      .color-menu-list {
+        .color-menu-list-item {
+          background: white;
+          height: 24px;
+          margin: 0;
+          padding: 0;
+          width: 24px;
+
+          .color-button {
+            background: white;
+            border: none;
+            border-radius: 0;
+            height: 24px;
+            margin: 0;
+            padding: 0;
+            width: 24px;
+
+            .color-swatch {
+              height: 12px;
+              margin: 6px;
+              width: 12px;
```

Comment:
Are these buttons the spec'ed size?  IMHO they seem very small.

Reply:
@bgoldowsky  Yeah, that's the spec'd size. FWIW, the `12px` width and height are just for the color swatch part of the button. There's also a border and outline for the swatch, plus an additional `6px` of padding on either side of the swatch to fill out the whole button. Does that still seem very small?

Reply:
To me, yes; I find it hard to really see the distinctions with such small patches.  But Michael's spec should govern here.  Not sure why he used different buttons here than in the other color-choosing menus in CLUE, but there is probably a reason.